pepdbagent/pepdbagent.py

```diff
@@ -45,47 +45,48 @@ def __init__(
         )
         sa_engine = pep_db_engine.engine
 
-        self.__sa_engine = sa_engine
+        self.pep_db_engine = pep_db_engine
+        self._sa_engine = sa_engine
 
-        self.__project = PEPDatabaseProject(pep_db_engine)
-        self.__annotation = PEPDatabaseAnnotation(pep_db_engine)
-        self.__namespace = PEPDatabaseNamespace(pep_db_engine)
-        self.__sample = PEPDatabaseSample(pep_db_engine)
-        self.__user = PEPDatabaseUser(pep_db_engine)
-        self.__view = PEPDatabaseView(pep_db_engine)
+        self._project = PEPDatabaseProject(pep_db_engine)
+        self._annotation = PEPDatabaseAnnotation(pep_db_engine)
+        self._namespace = PEPDatabaseNamespace(pep_db_engine)
+        self._sample = PEPDatabaseSample(pep_db_engine)
+        self._user = PEPDatabaseUser(pep_db_engine)
+        self._view = PEPDatabaseView(pep_db_engine)
 
```

Comment:
👍🏼 I like the look of the double underscore more... but its my understanding you aren't supposed to use double underscores...

Reply:
So, I agree with this change